json-lint/schemas/inventory/main.json

```diff
@@ -42,6 +42,9 @@
                 "adsense": {
                     "$ref": "./adservers/adsense.json"
                 },
+                "adSenseSticky": {
```

Comment:
here we map string --- ref  because like this we have autofill in vsCode, there is no other reason. 
Just remove the adSenseSticky, already we know is the adsense model with the previous key

Reply:
Fixed